install_compatibility_layer.sh

```diff
@@ -13,6 +13,13 @@ STORAGE=
 VERSION=2023.06
 VERBOSE=
 
+# RISC-V needs some different default values
+if [[ $(uname -m) = "riscv64" ]]; then
+  CONTAINER=docker://ghcr.io/eessi/bootstrap-prefix:debian-sid
+  REPOSITORY=riscv.eessi.io
+  VERSION=2024.03
```

Comment:
`20240307`

Reply:
Changed in https://github.com/EESSI/compatibility-layer/pull/202/commits/9463141c446c18735fcde9c303943f074e4293c5.